README.md

```diff
@@ -103,6 +103,28 @@ The classic running of an episode is therefore:
 - *action = model(obs_t)* and *env.step(action)*: execution of actions in the environment.
 - *env.close()*: end of simulation and destruction of the environment.
 
+### Training
+
+RL algorithms can be used with SofaGym to train RL agents for the provided environments to perform specific tasks. Currently, the training in SofaGym is done using Stable Baselines.
+
+To train a model for one of the registered environments of SofaGym, the ```rl.py``` script can be used:
+
+```bash
+python rl.py -e trunk-v0 -a PPO
+```
+
+Command line arguments can be used to change the training parameters:
+- -e, --env: environment name from available examples
+- -a, --algo: RL algorithm name from the algorithms supported by stable baselines
```

Comment:
It could be great to be able to list the names of all available algorithms

Reply:
included them
https://github.com/SofaDefrost/SofaGym/blob/a02a705c4d3b4fa88765b48c6ed5c0c602860349/README.md?plain=1#L115-L118